lib/postgres/postgres-pr/connection.rb

```diff
@@ -129,12 +129,101 @@ def peerport
     @conn.peerport
   end
 
+  def peerinfo
+    "#{peerhost}:#{peerport}"
+  end
+
   def current_database
     @params['database']
   end
 
-  def peerinfo
-    "#{peerhost}:#{peerport}"
+  # List of supported PostgreSQL platforms & architectures:
+  # https://postgrespro.com/docs/postgresql/16/supported-platforms
+  def map_compile_os_to_platform(compile_os)
```

Comment:
Taken from https://postgrespro.com/docs/postgresql/16/supported-platforms

I have bucket these by what I have covered versus not. Wanted to call this out in case anyone has thoughts or opinions.

Historical versions. Not covered and not present in `lib/msf/core/module/platform.rb`:
- 4.3BSD
- BEOS
- BSD/OS
- DG/UX
- Dynix
- NeXTSTEP
- QNX
- SCO
- SINIX
- Sprite
- SunOS 
- Tru64 UNIX 
- ULTRIX

Historical versions. Covered and present in `lib/msf/core/module/platform.rb`:
- HP-UX
- IRIX

17.6 supported but not covered in `lib/msf/core/module/platform.rb`:
- DragonFlyBSD
- illumos

17.6 supported and covered in `lib/msf/core/module/platform.rb`:
- Linux
- Windows
- Solaris
- macOS
- FreeBSD
- OpenBSD
- NetBSD
- AIX